docs/source/author/content_style_guide.rst

```diff
@@ -3,3 +3,32 @@ Content Style Guide
 
 .. contents:: Contents
   :local:
+
```

Comment:
This page is still pretty sparse, so I'll come up with suggestions for it #soon

Reply:
I really think this page needs to be expanded before #578 can be considered resolved

Reply:
I think it can still be resolved as it just says to create the page. It won't take much to fill it out a bit more in this PR though.